### ISSUE TITLE
All station stats access

### ISSUE THREAD

**Issue by MALEFOR27** [pmgl/starblast-modding#19]
There's no doubt that team mode is the most reliable root_mode to team based gameplay. However there's one rather annoying limitation and that's the station's statistics. People who've played U-Series, Nautics and Alien Intrusion all know how easy it is to make the enemy base leak crystals. It's often really easy to demolish a base in a single coordinated attack. Furthermore, with the mod I'm developing, this has been a brick wall in terms of balancing around my mod. 

So long story short, It would be very appreciated if mod creators could adjust the station shields capacity and other stats as they see fit.

**Comment by AlbieMorrison** [comment]
Yes, yes, and yes.
It would be much appreciated, and if there already is a way, could someone tell us?
